README.md

```diff
@@ -22,6 +22,10 @@
   <a href="https://www.npmjs.com/package/romcal/v/dev" target="_blank" rel="noopener noreferrer"><img alt="dev" src="https://img.shields.io/npm/v/romcal/dev?style=flat-square&logo=npm"></a>
 </p>
 
+<p align="center">
+  <a href="https://www.codetriage.com/romcal/romcal" target="_blank" rel="noopener noreferrer"><img alt="Code Triage Hellpers" src="https://www.codetriage.com/romcal/romcal/badges/users.svg" /></a>
```

Comment:
Could we use shields.io for badges please? Also there is a typo (`Hellpers`).

Reply:
whoopsies on the typo. i'll check about shields

Reply:
@tukusejssirs let me know if you'd like me to keep looking or do something else

Reply:
I think it won’t be possible to use shields.io, as we cannot get the codetriage users number (I could not find an API or something we could use) … unless we simply remove it.

Therefore, I think this is good to go. :pray: